### ISSUE TITLE
Allow Normalisers to produce raw files?

### ISSUE THREAD

**Issue by underdarknl** [minvws/nl-kat-coordination#277]
Currently, we have a one way flow of data from bytes into normalisers from where we expect the normaliser to output Octopoes model compliant OOI's, this works for most things, but There's a nother option.

What is normalisers can also consume raw files, and then return raw files.
This could be useful in the context of packaged files. Files like zip's, container or VM images etc contain other fragments which in turn might already have normalisers available.
However, those normalisers will not be started trough the normaliser extracting the zip, since we already have that logic somewhere else. I propose we chart what needs to be done to allow normalisers to output raw files back to bytes again, and what this means for the frontend, the scheduler and chain of custody leading from the original raw file into the eventual OOI's.

**Comment by noamblitz** [comment]
Very interesting and could make a lot of sense. Just doubting if we should let normalizer have this as a possibility versus storing the whole zip/whatever in octopoes and letting bits unpack them for example.

**Comment by underdarknl** [comment]
Yes, I've thought about that too, but keeping raw data in octopoes seems like a step too far. Only store the relevant meta-data about the zip in octopoes (as a file ooi or something), and then have the normalizer create new raw files next to the creation of that file OOI. Another normalizer (which we might already have) will then pick up those newly created Raw files and process them into more specific OOI's if there's anything in there that we understand.

**Comment by stephanie0x00** [comment]
Discussionmeet: we will use stdin/stdout
